### ISSUE TITLE
Gmail: Cannot select checkbox to skip & archive when creating a filter

### ISSUE THREAD

**Issue by orschiro** [philc/vimium#3128]
Dear all, 

Can someone please try to reproduce whether he or she can select the checkbox to skip and archive an email when creating a filter in Gmail?

I have created the following video for you to reproduce it.

Video: https://drive.google.com/file/d/1-11RPKqH-kFkoQDxK28AqJ8S7kxMZV-s/view

Thank you!

Yours,

Robert

**Comment by smblott-github** [comment]
Confirmed.

**Comment by orschiro** [comment]
Thanks Stephen!

Is that something that can be fixed by Vimium?

Yours,

Robert

**Comment by gdh1995** [comment]
@smblott-github This bug is because there was a time, that:
1. Angular Material used `mouseEvent.clientX` and `.clientY` to draw an animation where a dark circle zoomed in
2. related `click` callbacks would only be called when the animation above was finished
3. but for Vimium, both `clientX` and `clientY` are always 0, which causes a bug of Material when computing the delay
4. so no work functions would be called

And a simple solution is to use `DomUtils.getVisibleClientRect` to get a rect and compute its center.

**Comment by orschiro** [comment]
> Updated: I found the issue on 2018/2/15, but a latest Material seems to have fixed / worked around this.

@gdh1995 what does that mean? Can we close this issue?

**Comment by gdh1995** [comment]
No. I just mean I haven't found a classical Material webpage still having such an issue - Google has compressed (uglified) used libraies and I can not find the related lines easily.
